client-src/public/script.js

```diff
@@ -0,0 +1,55 @@
+$(document).ready(function() {
+  // $('.select').material_select();
+  $(".button-collapse").sideNav();
+  $(".tooltipped").tooltip({ delay: 10 });
+  const documentHeight = $(document).innerHeight();
+  const contentHeight = $(".mfl-page-wrap").innerHeight();
+  const marginDiff = documentHeight - contentHeight;
+
+  $(document).tooltip();
+
+  if ($("#back-to-top").length) {
+    var scrollTrigger = 100, // px
+      backToTop = function() {
+        var scrollTop = $(window).scrollTop();
+        if (scrollTop > scrollTrigger) {
+          $("#back-to-top").addClass("show");
+        } else {
+          $("#back-to-top").removeClass("show");
+        }
+      };
+    backToTop();
+    $(window).on("scroll", function() {
+      backToTop();
+    });
+    $("#back-to-top").on("click", function(e) {
+      e.preventDefault();
+      $("html,body").animate({ scrollTop: 0 }, 700);
+    });
+  }
+
+  const content = $(".content");
+  const nav = $(".header");
```

Comment:
Variable naming, better off with something like header because you are pointing to the .header element 😎